### ISSUE TITLE
Create an initial app navigation

### ISSUE THREAD

**Issue by Rupak-B** [CSAllenISD/2023-ISP-SchoolFood#11]
After figuring out what we need to do for a basic viable project we need to create a design

- What pages do we need
- How easily navigate pages
- Easy to read user interface

**Comment by brett-k-cs** [comment]
Correct. I say we take the Mock App and convert that into an App with only navigation working with static restaurants / food
